Server/src/usercoordinator.erl

```diff
@@ -0,0 +1,69 @@
+-module(usercoordinator).
+-export([init/1]).
+
+%%Author: Tim Jonasson
+%%Version: 1
+
+%Initializes the usercoordinator
+init(Socket) -> 
+  inet:setopts(Socket, [{active, true}]),
+  loop(Socket, []).
+
+  
+loop(Socket, Diagrams) -> 
+  receive
+    {tcp, Port, Info} -> 
```

Comment:
Is it all one receive clause?

Reply:
Yes, do you want me to try and divide it up a bit to make it easier to read?

Reply:
If possible then yes. Otherwise you can just clarify that point through comments.